src/geshi/lisp.php

```diff
@@ -52,30 +52,301 @@
-            'lognand','lognor','logorc2','logtest','logbitp','logcount',
-            'integer','nil','parse-integer','make-list','print','write'
+            '*', '**', '***', '+', '++', '+++', '-', '/', '//', '///',
+            '/=', '1+', '1-','<', '<=', '=', '>', '>=', 'abort', 'abs',
+            'access', 'acons','acos', 'acosh', 'add-method', 'adjoin', 
+            'adjust-array','adjustable-array-p', 'allocate-instance', 
+            'alpha-char-p','alphanumericp', 'and', 'append', 'apply', 
+            'applyhook','apropos', 'apropos-list', 'aref', 
+            'arithmetic-error','arithmetic-error-operands', 
+            'arithmetic-error-operation','array', 'array-dimension', 
+            'array-dimension-limit','array-dimensions', 
+            'array-displacement','array-element-type', 
+            'array-has-fill-pointer-p','array-in-bounds-p', 
+            'array-rank','array-rank-limit', 'array-row-major-index', 
+            'array-total-size','array-total-size-limit', 'arrayp', 
+            'ash','asin', 'asinh', 'assert', 'assoc', 'assoc-if', 
+            'assoc-if-not','atan', 'atanh', 'atom', 'backquote', 
+            'baktrace','base-char', 'base-string', 'bignum', 
+            'bignums','bit', 'bit-and', 'bit-andc1', 'bit-andc2', 
+            'bit-eqv','bit-ior', 'bit-nand', 'bit-nor', 'bit-not', 
+            'bit-orc1','bit-orc2', 'bit-vector', 'bit-vector-p', 
+            'bit-xor','block', 'boole', 'boole-1', 'boole-2', 
+            'boole-and','boole-andc1', 'boole-andc2', 'boole-c1', 
+            'boole-c2','boole-clr', 'boole-eqv', 'boole-ior', 
+            'boole-nand','boole-nor', 'boole-orc1', 'boole-orc2', 
+            'boole-set','boole-xor', 'boolean', 'both-case-p', 
+            'boundp','break', 'broadcast-stream', 
+            'broadcast-stream-streams','built-in-class', 'butlast', 
+            'byte','byte-position', 'byte-size', 'caaaar', 'caaadr', 
+            'caaar','caadar', 'caaddr', 'caadr', 'caar', 'cadaar', 
+            'cadadr','cadar', 'caddar', 'cadddr', 'caddr', 'cadr', 
+            'call-arguments-limit','call-method', 'call-next-method', 
+            'capitalize','car', 'case', 'catch', 'ccase', 'cdaaar', 
+            'cdaadr','cdaar', 'cdadar', 'cdaddr', 'cdadr', 'cdar', 
+            'cddaar','cddadr', 'cddar', 'cdddar', 'cddddr', 'cdddr', 
+            'cddr','cdr', 'ceiling', 'cell-error', 'cell-error-name', 
+            'cerror','change-class', 'char', 'char-bit', 'char-bits', 
+            'char-bits-limit','char-code', 'char-code-limit', 
+            'char-control-bit','char-downcase', 'char-equal', 
+            'char-font','char-font-limit', 'char-greaterp', 
+            'char-hyper-bit','char-int', 'char-lessp', 
+            'char-meta-bit','char-name', 'char-not-equal', 
+            'char-not-greaterp','char-not-lessp', 'char-super-bit', 
+            'char-upcase','char/=', 'char<', 'char<=', 'char=', 
```

Comment:
Symbols like < and = will likely cause havoc in the keyword section. Please avoid.